pkgs/top-level/aliases.nix

```diff
@@ -637,6 +637,7 @@ mapAliases {
   libyamlcpp_0_3 = yaml-cpp_0_3; # Added 2023-01-29
   lightdm_gtk_greeter = lightdm-gtk-greeter; # Added 2022-08-01
   lightstep-tracer-cpp = throw "lightstep-tracer-cpp is deprecated since 2022-08-29; the upstream recommends migration to opentelemetry projects.";
+  limesctl = throw "limesctl has been removed because it is insignificant."; # Added 2024-11-05
```

Comment:
Can we come up with something that makes more sense... Imagine you are someone updating the system and see something is removed because "it is insignificant"